svthalia/concrexit
# Added banner with reminder for uploading your profile picture.

Closes Show a banner for members that have an empty profile #2256



### Summary
Added a banner on the top of the screen. that appears when members don't have profile picture yet. The banner goes away once a profile picture has been uploaded.

### How to test
Steps to test the changes you made:
1. Delete your profile picture if you have one.
2. Now the banner should be showed on the top of the screen, telling you to upload a picture.
3. Click on the link, it should redirect you to the profile editing page.
4. Upload a profile picture.
5. Now the banner should be gone.
